script.js

```diff
@@ -0,0 +1,32 @@
+const words = [
+    APPLE,
+    ORANGE
+]
+
+//Save and transform to upper case the letter the user wrote:
+function tryLetter(){
+    let letter = document.getElementById('letter').value
+    let LETTER = letter.toUpperCase()
+}
+
+//Write the letter if correct, mark as a mistake if wrong:
+for(let i=0; i<words.length; i++){
+    if (LETTER === words[1][i]) {
```

Comment:
Here I see this if condition:
`LETTER === words[1][i]`

I need to understand better, what would you like for this condition to check exactly?

Reply:
If the letter the person typed (LETTER) is the same as any letter of the first word on the array (words[1][i]) put the letter on the exact place (like a hangman) if not, you made a mistake.

Reply:
Ok, I think here is where we can practice some problem-solving thinking and breaking down a problem into steps.
I can suggest some ideas on how to move forward with this, but you can always of course come up with your own approach on how to move further.
I think you could use the split() method and split the word into single letters
Split will create an array of letters from the word you are splitting
So then you can use a for loop to compare each letter in the word to the letter the person has typed
Maybe the letters have a CSS styling that they are initially hidden (display: none)
But if the letters match in the for loop check you are doing, then maybe you change the CSS to display the letter that matched
How does that sound?